### PR TITLE
Post for updates in Okta Angular SDK

### PR DESCRIPTION
**Pull request by alisaduncan**
This blog post has the following features:

- [X] A title that's approved by Dev Advocacy
- [X] A URL that's approved by Dev Advocacy
- [X] The content has been run through Grammarly (https://www.grammarly.com/)
- [X] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

**Review by edunham — APPROVED** [review]
Short, sweet, and to the point -- beautiful!

File: _source/_posts/2023-02-20-angular-forroot.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+Since authentication-related services should be application-wide, we recommend using the `OktaAuthModule.forRoot()` static method when configuring Okta in new Angular applications. You're still covered with backward-compatibility support if you have more complex needs, such as requiring a factory method to provide the `OKTA_CONFIG` injection token.
+
+The Okta Angular SDK team plans to improve this in the future. They recognize the current `OktaAuthModule` can improve, and to truly support `forRoot()`, they will have to remove auth services from the `providers` array of the module, which breaks backward compatibility. The Okta Angular SDK team also is preparing to support the latest Angular features, which will become the Angular recommended patterns going forward, such as stand-alone components, module-less architecture, and functional route guards. If you have thoughts or input, or want to share how you use the Okta Angular SDK so the team can make sure they consider it, feel free to drop a note in the comment below.
+
+## Learn more about Angular, Dependency Injection, and OpenID Connect
+
+If you liked this post, you might want to check out the following.
```
Comment (edunham) [inline _source/_posts/2023-02-20-angular-forroot.md]:
```suggestion
If you liked this post, you might want to check out the following:
```

File: _source/_posts/2023-02-20-angular-forroot.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        OktaAuthModule 
+    ],
+    providers: [
+        { provide: OKTA_CONFIG, useValue: {oktaAuth} }
+    ]
+})
+export class AppModule { }
+```
+
+This method will continue to work, and you can continue using the module as you have it now without any changes.
```
Comment (edunham) [inline _source/_posts/2023-02-20-angular-forroot.md]:
```suggestion
This method will continue to work, and you can keep on using the module as you have it now without any changes.
```

**Review by edunham — APPROVED** [review]
Demonstrating editorial changes to show Colton the github review process :)

File: _source/_posts/2023-02-20-angular-forroot.md (hunk 2 above)
Comment (alisaduncan) [reply _source/_posts/2023-02-20-angular-forroot.md]: @edunham maybe a fine line, but I'd like to keep it as continue. Trying to encourage the pattern going forward should change, and keep on sounds more positive than continue to me?

**Pull request merged**